packages/plugin-build-node/src/index.ts

```diff
@@ -40,7 +40,7 @@ export async function build({out, reporter}: BuilderOptions): Promise<void> {
             babelPresetEnv,
             {
               modules: false,
-              targets: {node: '6'},
+              targets: {node: options.minNodeVersion || '6'},
```

Comment:
As the `6` is subject to change, it would be nice to have it defined as a `DEFAULT_MIN_NODE_VERSION` constant, on top of the file, to avoid magic numbers. No big deal, tho!

Reply:
added